### PR TITLE
doc: fix markdown errors in api-plugin doc

### PR DESCRIPTION
**Pull request by seeflood**
**What this PR does**:
Fix markdown errors in `api_plugin/design`
image

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by stulzq — DISMISSED** [review]

**Comment by seeflood** [comment]
now it's ready for review:
image

File: docs/zh/design/api_plugin/design.md [hunk 1]
```diff
@@ -1,54 +1,65 @@
...
+3. 用户想给 Dapr API 加字段，先在自己的 Fork 版本里添加了字段、满足线上需求，然后将 PR 提给社区。社区拒绝添加该字段，PR 被关闭。用户很尴尬：这字段已经在线上使用了，怎么处理？
 
 ## 2. High level design
+
 ### 2.1. Hierarchical API
-参考OS领域当年是怎么定API的，我们可以把Runtime API设计成多层：
 
+参考 OS 领域当年是怎么定 API 的，我们可以把 Runtime API 设计成多层：
 
 ![img.png](../../../img/api_plugin/img.png)
```
Comment (stulzq) [inline docs/zh/design/api_plugin/design.md:49]: image err

Comment (seeflood) [reply docs/zh/design/api_plugin/design.md:49]: I think it's a flaw of github preview feature. The doc site is ok:
image

Anyway, I will upload this image to a CDN

Comment (seeflood) [reply docs/zh/design/api_plugin/design.md]: @stulzq fixed.Please review again

**Review by zhenjunMa — APPROVED** [review]

**Review by wenxuwan — APPROVED** [review]

**Pull request merged**